cadccutout/cadccutout/transform.py

```diff
@@ -76,16 +76,16 @@
 
 from astropy import units as u
 from astropy.coordinates import SkyCoord, Longitude, Latitude, ICRS
-from astropy.wcs import WCS
+from astropy.wcs import WCS, NoConvergence
 from regions.shapes.circle import CircleSkyRegion
 from regions.shapes.polygon import PolygonSkyRegion
 
 from .no_content_error import NoContentError
 from .pixel_cutout_hdu import PixelCutoutHDU
 from .shape import Circle, Polygon, Energy, Time, Polarization, \
-                   PolarizationState
+    PolarizationState
 
-logger = logging.getLogger(__name__)
+# logger = logging.getLogger(__name__)
```

Comment:
Uncomment out. This is the logger for 'cutout'. If you want to set it to debug level in test files you can do: logging.getLogger(cadccutout.__name__).setLevel(logging.DEBUG).